index.js

```diff
@@ -304,6 +305,8 @@ Truth.prototype.clone = function clone(name) {
   var truth = new Truth(name);
 
   truth.transforms = this.transforms.slice();
+  truth.unique = this.unique;
+
```

Comment:
I already added it but forgot to push it. I merged, rebased, tested and published it

Arnout Kazemier

> On Jul 9, 2015, at 8:26 PM, Martijn Swaagman <notifications@github.com> wrote:
> 
> In index.js:
> 
> > @@ -304,6 +305,8 @@ Truth.prototype.clone = function clone(name) {
> >    var truth = new Truth(name);
> >  
> >    truth.transforms = this.transforms.slice();
> > +  truth.unique = this.unique;
> > +
> Didn't want to provide clone with an options arg here as I thought it looked silly. But yeah that would def be an improvement
> 
> —
> Reply to this email directly or view it on GitHub.
>